### PR TITLE
chore: update iOS workflows

### PR DESCRIPTION
**Pull request by bohdanprog**
# Summary

Update iOS workflows.

### CODE REVIEW

File: .github/workflows/ios-e2e.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     timeout-minutes: 60
     env:
       WORKING_DIRECTORY: example
-      DEVICE: iPhone 14 Pro
+      DEVICE: iPhone 16 Pro
```
Comment (jakex7) [inline .github/workflows/ios-e2e.yml:22]: In E2E tests, `iPhone 14 Pro` was chosen intentionally, as it has the exact pixel ratio needed

**Review by jakex7 — APPROVED** [review]
Left one comment, otherwise LGTM

File: .github/workflows/ios-e2e.yml (hunk 1 above)
Comment (bohdanprog) [reply .github/workflows/ios-e2e.yml:22]: Don't they have the same pixel ratio?

Comment (bohdanprog) [reply .github/workflows/ios-e2e.yml:22]: @jakex7

**Pull request merged**